jdaviz/configs/default/plugins/export/export.vue

```diff
@@ -152,7 +171,7 @@
              :results_isolated_to_plugin="true"
              @click="interrupt_recording"
              :disabled="!movie_recording"
-          >
+          traitlets.String>

```

Comment:
Out of curiosity (I have much to learn in vue), what does this do?

Reply:
this was a mistake i think!